### ISSUE TITLE
SQL templates to be all lowercase

### ISSUE THREAD

**Issue by joshenlim** [supabase/supabase#5753]
# Chore

Decided from here https://github.com/supabase/supabase/pull/5739 that we'd adhere to a practice of standardizing the way we write SQL for our users on the dashboard ([ref](https://github.com/mattm/sql-style-guide/blob/master/README.md))

Matter of preference really, but just a decision to stick to one

**Comment by nwneisen** [comment]
Hello :wave:

I'm interested in contributing to the project and this issue looked like a good first issue for me. I'd like to claim it and verify I understand everything that will be needed.

1. Update the SQL statements found in `studio/components/interfaces/SQLEditor/SQLEditor.constants.ts`
2. Follow the SQL conventions found in this [style guide](https://github.com/mattm/sql-style-guide/blob/master/README.md)
3. Run studio and verify the statements are formatted correctly

Looking forward to joining in on the project!

**Comment by Isaiah-Hamilton** [comment]
Hey @nwneisen - I've assigned you to this issue and yes everything that you've listed is correct. Can't wait to see your pull request and become a contributor.

**Comment by nwneisen** [comment]
I've opened a PR for this change. There were some statements that weren't explicitly mentioned in the style guide that I used my best judgement to do something similar to other statements in the style guide. 

I had an issue setting up Supabase for local development and I wasn't able to see the statements in the UI. My setup only had a default project and it wouldn't let me view the SQL statements the way https://app.supabase.io/ does. Is there a developer setup doc that I might have missed?
